lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -730,7 +734,8 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
                 header = [section['label'] for section in gradeset[u'section_breakdown']]
                 rows.append(
                     ["id", "email", "username", "grade"] + header + cohorts_header +
```

Comment:
Nit (and not your code): this seems a strange idiom of mixing ``+`` and ``append`` to append a number of rows. Should it be using ``rows.extend`` instead?

http://stackoverflow.com/questions/16621498/how-to-append-multiple-items-in-one-line-in-python